tsconfig.base.json

```diff
@@ -1,7 +1,7 @@
 {
   "compilerOptions": {
     "strict": true,
-    "target": "es6",
+    "target": "es2020",
```

Comment:
this is a perf fix, because async/await no longer become generators

Reply:
Can you please move this to a separate pull request?

Reply:
> Can you please move this to a separate pull request?

yep - it's part of 1780, so I think we can just let that land (it sets esnext, which i think should be ok?)

Reply:
I believe so!